assets/models/system/bytetrack_yolox_x_crowdhuman_mot17-private-half/description.md

```diff
@@ -8,13 +8,13 @@ Multi-object tracking (MOT) aims at estimating bounding boxes and identities of
 
 Inference type|Python sample (Notebook)|CLI with YAML
 |--|--|--|
-Real time|<a href="https://aka.ms/azureml-video-mutli-object-tracking-online-inference" target="_blank">video-multi-object-tracking-online-endpoint.ipynb</a>|[video-multi-object-tracking-online-endpoint.sh](https://github.com/Azure/azureml-examples/blob/yiyoulin/mot_examples/cli/foundation-models/system/inference/video-multi-object-tracking/video-multi-object-tracking-online-endpoint.sh)|
+Real time|<a href="https://aka.ms/azureml-video-mutli-object-tracking-online-inference" target="_blank">video-multi-object-tracking-online-endpoint.ipynb</a>|[video-multi-object-tracking-online-endpoint.sh](https://aka.ms/cli-video-multi-object-tracking-online-inference)|
```

Comment:
Can we use the same format for hyperlink for cli example as well. <a href...></a>

Reply:
Same for ocsort cases.